mkdocs.yml

```diff
@@ -24,11 +24,13 @@ markdown_extensions:
   - toc:
       permalink: true
   # pymdownx
+  - pymdownx.escapeall
   - pymdownx.highlight:
       anchor_linenums: true
   - pymdownx.superfences
 nav:
   - index.md
+  - changelog.md
```

Comment:
I would probably move this to the end of the list and adjust the title to "Changelog of antsibull-changelog". WDYT?

Reply:
ping @gotmax23

Reply:
See the latest push